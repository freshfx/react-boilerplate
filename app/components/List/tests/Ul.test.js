import 'jest-styled-components'
import React from 'react'
import {mount} from 'enzyme'

import Ul from '../Ul'

const renderComponent = (props = {}) => mount(<Ul {...props} />)

describe('<Ul />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderComponent()
    expect(renderedComponent).toMatchSnapshot()
  })
})
