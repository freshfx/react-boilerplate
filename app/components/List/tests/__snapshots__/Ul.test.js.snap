// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ul should match snapshot 1`] = `
.emotion-0 {
  list-style: none;
  margin: 0;
  width: 100%;
  max-height: 30em;
  overflow-y: auto;
  padding: 0 1em;
}

<ul
  class="emotion-0 emotion-1"
/>
`;
